docs/release-notes/2.25.0.md

```diff
@@ -0,0 +1,84 @@
+
+
+- **Enhanced PostgreSQL monitoring**
+    - PMM will now let users who add PostgreSQL Servers be monitored by PMM to [specify database name for monitoring](https://deploy-preview-643--pmm-doc.netlify.app/details/commands/pmm-admin.html#postgresql). Previous versions have the default `postgres` database, which wasn't always desired. 
+
+    -  Now that `pg_stat_monitor` has released it's [Release Candidate v1.0.0-rc.1](https://github.com/percona/pg_stat_monitor/releases/tag/1.0.0-rc.1), it is fully supported in PMM from this release forward with many new PostgreSQL metrics, Dashboards and Query Analytics features available!  You can read more about its features and advantages in this version from [pg_stat_monitor User guide](https://github.com/percona/pg_stat_monitor/blob/1.0.0-rc.1/docs/USER_GUIDE.md) 
+
+    - PMM 2.25.0 is also now compatible with the latest Percona Distributions for PostgreSQL [14](https://www.percona.com/doc/postgresql/LATEST/release-notes-v14.1.upd2.html), [13](https://www.percona.com/doc/postgresql/13/release-notes-v13.5.upd2.html), [12](https://www.percona.com/doc/postgresql/12/release-notes-v12.8.upd.html), [11](https://www.percona.com/doc/postgresql/11/release-notes-v11.13.upd.html) updated on December 7, 2021 , which includes the newest version of pg_stat_monitor extension.  
+
+
+
+- **Grafana usability enhancements**
+    - We changed the way that dashboards, plugins, and data sources are added into PMM. We are now using the native Grafana provisioning mechanism in PMM. This change will lead to a faster and more reliable update process for our users moving from one version of PMM to another. With this update, we'll preserve all custom changes you have already done with plugins and dashboards, but we still encourage you to back them up before an upgrade and recheck after, just in case. 
+
+    - This version of PMM also will let you change the time zone on dashboards. It will also preserve this selection while you navigate over Dashboards. If you need to change this setting permanently for your account - you need to change it in your preferences by the URL: `https://YOUR_PMM_SERVER/graph/profile`
+
+- **DBaaS (Technical Preview)**
+    - You can now deploy and update your DBaaS created PXC clusters to the latest version of Percona Distribution for MySQL Operator 1.9.0 to take advantage of the latest features and fixes. 
+
+- **PMM environment enhancements**
+    - For users who use PMM client in kubernetes or build automation around it, we added a pmm-client docker container to be started as a sidecar container and not fail if the connection to DB  is not available. Please see full details in [PMM Client documentation](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/client/index.html#install)

```

Comment:
```suggestion
    - Added a pmm-client docker container relevant if you're using PMM client in Kubernetes or if you build automation around it.
This can be started as a sidecar container and will not fail if the connection to DB  is not available. For more details, see the [PMM Client documentation](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/client/index.html#install)

```

Reply:
the pmm-client docker was already in PMM. What we added - is an additional flag, so people can start it as sidecar

Reply:
```suggestion
     - The pmm-client docker container can now be started as a sidecar.  For users that use PMM client in Kubernetes or build automation around it, you can now start the client as a sidecar container.  The client will also gracefully handle any instances where the connection to DB  is not available. For more details, see the [PMM Client documentation](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/client/index.html#install)
```